bin/import

```diff
@@ -13,20 +13,14 @@ print_error() {
 }
 
 curl -O https://data.gov.uk/data/dumps/data.gov.uk-ckan-meta-data-latest.organizations.jsonl.gz
-curl -O https://data.gov.uk/data/dumps/data.gov.uk-ckan-meta-data-latest.v2.jsonl.gz
-
 gunzip data.gov.uk-ckan-meta-data-latest.organizations.jsonl.gz
-gunzip data.gov.uk-ckan-meta-data-latest.v2.jsonl.gz
 
 print "Adding locations"
 rails import:locations["lib/seeds/locations.csv"]
 
 print "Adding organisations"
 rails import:legacy_organisations["data.gov.uk-ckan-meta-data-latest.organizations.jsonl"]
 
-print "Adding datasets"
-rails import:legacy_datasets["data.gov.uk-ckan-meta-data-latest.v2.jsonl"]
-
 print "Cleaning up"
 rm data.gov.uk-ckan-meta-data-latest.organizations.jsonl data.gov.uk-ckan-meta-data-latest.v2.jsonl
```

Comment:
`data.gov.uk-ckan-meta-data-latest.v2.jsonl` can be removed from this cleanup step too.

Reply:
Done - thanks for spotting :-)